docs/rules/max-statements.md

```diff
@@ -1,4 +1,4 @@
-# Limit Maximum Number of Statements (max-statements)
+# enforce a maximum number of statements allowed in `function` blocks (max-statements)
```

Comment:
Let’s also add this rule description for future consideration.

How about replace `function` with function in the body of the doc, because it applies to arrow functions too?